### PR TITLE
Notify admin 931 add a preferred timezone widget

### PR DESCRIPTION
**Pull request by terrazoon**
Add a selector widget so the user can change their preferred timezone.  Offer choices in the main US timezones that should handle daylight/standard automatically.

### CODE REVIEW

File: app/templates/views/user-profile.html [hunk 1]
```diff
@@ -63,6 +63,17 @@ <h1 class="font-body-2xl margin-bottom-3">User profile</h1>
           )
         }}
       {% endcall %}
+      IS IT HERE???? {{ current_user.preferred_timezone }}
```
Comment (ccostino) [inline app/templates/views/user-profile.html:66]: It looks like this is a debug statement that got left in.

```suggestion
```

**Review by ccostino — COMMENTED** [review]
Thanks, @terrazoon!

It looks like there was a debug print statement left in the template.  Also, I'm noting that we'll need a DB data migration to actually set a default timezone for all existing users as I didn't have one set, which prevented me from going into a service (an `UnknownTimeZoneError` error was thrown because my user object had none).

Alternatively, we could have the UI do this for the user automatically the first time they go to a service and if no preference is detected.  We could have it default to `US/Eastern` and just have a flash message appear saying that the user had no timezone set so it defaulted to `US/Eastern` and that if they'd like to change it to go to their user settings.

File: app/templates/views/user-profile.html (hunk 1 above)
Comment (terrazoon) [reply app/templates/views/user-profile.html]: This actually was here.  I just removed it.  There is a bug in GitHub where sometimes reviews don't update to the latest pushed commit, but it didn't happen this time.  This time was developer error.

As for handling the lack of the default "US/Eastern", that was addressed in a migration in api project separately.

**Review by ccostino — APPROVED** [review]
Thanks, @terrazoon!

File: app/templates/views/user-profile.html (hunk 1 above)
Comment (ccostino) [reply app/templates/views/user-profile.html]: Awesome, thank you again!

**Pull request merged**